x/streamer/client/cli/tx_pause_stream.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/dymensionxyz/dymension/x/streamer/types"
+
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+)
+
+// NewCreateStreamCmd broadcasts a CreateStream message.
+func NewCmdSubmitStopStreamProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop-stream-proposal streamID [flags]",
+		Short: "proposal to stop an exisiting stream",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			streamID, err := strconv.ParseUint(args[0], 10, 64)
```

Comment:
prob worth checking also if the stream exists before a proposal is sent (also applied to tx_replace and tx_update proposals)

Reply:
it doesn't have access to the app in this stage. it's just creates the proposal message actually.
the validation logic happens in the handler